api/tls.go

```diff
@@ -0,0 +1,37 @@
+package api
+
+import (
+	"fmt"
+)
+
+// TLS is the structure of an app's TLS settings.
+type TLS struct {
+	fmt.Stringer
+	// Owner is the app owner. It cannot be updated with TLS.Set(). See app.Transfer().
+	Owner string `json:"owner,omitempty"`
+	// App is the app name. It cannot be updated at all right now.
```

Comment:
Maybe somethng like "App is the app the tls settings apply to" instead? I'd rather make it obvious that TLS is a child of app rather than just say that it can't be updated.